node.go

```diff
@@ -127,20 +128,38 @@ func (n *node) get(
 
 // finalize current just returns the node count. However, it will prune the
```

Comment:
Should we update this?

Reply:
Thanks! I updated this and force-pushed.